ovs/lib/albacontroller.py

```diff
@@ -251,7 +251,10 @@ def add_cluster(alba_backend_guid):
 
         etcd_key = AlbaController.ETCD_NR_OF_AGENTS_KEY.format(alba_backend_guid)
         nr_of_storage_nodes = len(AlbaNodeList.get_albanodes())
-        EtcdConfiguration.set(etcd_key, nr_of_storage_nodes)
```

Comment:
The setup still does that, but at this point, at the node being installed, the asd-manager setup has not been executed yet. So basically, the correct thing to do here is make sure that the AlbaController.add_cluster also (at the end) runs that one method that does the node detection.